include/zen/screen-layout.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <wlr/util/box.h>
+
+struct zn_scene;
+
+struct zn_screen_layout {
+  struct zn_scene *scene;
+  struct wl_list screens;  // zn_screen::link
```

Comment:
With the current convention
- `*_list` for wl_list
- `*s` for wl_array

```suggestion
  struct wl_list screen_list;  // zn_screen::link
```